HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -183,11 +189,19 @@ def customiseFor29512(process):
 
     return process
 
+# Use 8p fit (PR29617) and don't apply HB- correction (PR26177)
+def customiseFor29617(process):
+    for producer in producers_by_type(process, "HBHEPhase1Reconstructor"):
+        producer.algorithm.chiSqSwitch = cms.double(-1.0)
+        producer.algorithm.applyLegacyHBMCorrection = cms.bool( False )
+    return process
```

Comment:
I was puzzled at first by the change in the plots in the HI Run2 workflow 
https://cmsweb.cern.ch/dqm/dev/start?runnr=1;dataset%3D/RelVal_wf158_0_pr/CMSSW_11_1_X-PRcmssw_29617-6107/DQMIO;referenceshow%3Dall;referencenorm=False;referenceobj1%3Dother::/RelVal_wf158_0_base/CMSSW_11_1_X-PRcmssw_29617-6107/DQMIO::;sampletype%3Doffline_relval;workspace%3DEverything;

The changes in this file are not here to patch the config to keep the HLT running.
Instead  these are to change its default behavior. Isn't this a wrong way to do it? 
If the goal is to change existing parameters in the existing tables, shouldn't there be a JIRA ticket instead?

Reply:
@slava77 
* synchronizeHCALHLTofflineRun3on2018data and synchronizeHCALHLTofflineRun2 are function to let now play and testing in old data/menu
* customiseFor29617: is for the Run3 menu

For a modification in the confDB we usually propose a change in a customization function like this, then @Martin-Grunewald/TSG import the modification in the ConfDB in the Run3 menu and dump the menu the release. A JIRA is also needed to activate TSG. Will do.

Reply:
> customiseFor29617: is for the Run3 menu

however, it apparently applies to the Run2 menu used in wf 158.0